src/createDIContainer.js

```diff
@@ -53,7 +55,8 @@ async function createDIContainer() {
    * Config
    */
   container.register({
-    configFilePath: asValue(path.resolve(__dirname, '../data/config.json')),
+    dataDirRepository: asClass(dataDirRepository),
+    configFilePath: asValue(path.resolve(os.homedir(), '.mn/config.json')),
```

Comment:
I think we don't need it since we have `homeDirPath`. I would allow people to connfigure home dir path with `MN_HOME_DIR_PATH` env variable.

Reply:
You mean for running standalone docker-compose without the CLI? Why does it need access to ~/.mn/config.json, it can't read this file, all config is taken from envs. Don't really understand what you mean here.